tests/Api/Shop/OrdersTest.php

```diff
@@ -78,13 +78,33 @@ public function it_gets_order_items(): void
 
         $this->addItemToCart('MUG_BLUE', 3, $tokenValue);
 
-
         $this->client->request('GET', '/api/v2/shop/orders/nAWw2jewpA/items', [], [], self::CONTENT_TYPE_HEADER);
         $response = $this->client->getResponse();
 
         $this->assertResponse($response, 'shop/get_order_items_response', Response::HTTP_OK);
     }
 
+    /** @test */
+    public function it_returns_nothing_if_visitor_tries_to_get_the_order_items_of_logged_in_user(): void
```

Comment:
```suggestion
    public function it_returns_nothing_if_visitor_tries_to_get_the_items_of_a_user_order(): void
```
technically there is no such thing as a logged-in user in the API, furthermore, this naming insinuates that if a user logs out, then visitors can access it.